hub/gads-ui/src/components/DeviceControl/Tabs/Actions/Apps/UninstallApp.js

```diff
@@ -1,7 +1,7 @@
 import { Box, Button, CircularProgress, FormControl, MenuItem, Select, Stack } from '@mui/material'
 import InstallMobileIcon from '@mui/icons-material/InstallMobile'
 import './InstallApp.css'
-import { useState } from 'react'
+import { useEffect, useState } from 'react'
```

Comment:
`useEffect` is imported but not used

Reply:
Done [here](https://github.com/shamanec/GADS/pull/185/commits/4957832b1908943c42adb380c3ebcb9b9852e2b1)